proto/executor.proto

```diff
@@ -33,9 +35,30 @@ message Executor {
 	optional Aggregation aggregation = 5;
 	optional TopN topN = 6;
 	optional Limit limit = 7;
-	optional Aggregation stream_agg= 8;
+	optional ExchangeReceiver exchange_receiver = 8;
```

Comment:
Why overwrite `stream_agg`?

Reply:
Because it's unnessary and never used. We'd better remove the garbage as soon as we can.